### PR TITLE
Clean up "mess"

### PR DESCRIPTION
**Pull request by ghost**
I may have forgot to modify some files. =

### CODE REVIEW

File: Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/Mod.java [hunk 1]
```diff
@@ -165,7 +165,7 @@ public final void setEnabled(boolean enabled)
 	{
 		this.enabled = enabled;
 		active = enabled && !blocked;
-		if(blocked && enabled)
+		if((blocked && enabled) || this.isEnabled() == enabled)
```
Comment (liach) [inline Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/Mod.java]: ?

**Comment by ghost** [comment]
Encoding is broken, it has to be `Windows-1252` ([Reference: Alexander01998](https://github.com/Wurst-Imperium/Wurst-Client/pull/2393#issuecomment-190793615)) 
10ec46

In this image the encoding is ISO-8859-1 because ISO-8859-1 (Reference: Me, GitHub) and ANSII ([Reference: wkuba500](https://github.com/Wurst-Imperium/Wurst-Client/pull/2398#issuecomment-192633404)) works too. If you can't change the encoding, [do you want me to fix it](https://github.com/s3r1alpari4h/Wurst-Client-for-MC-1.9.X/pull/1)?

**Comment by Alexander01998** [comment]
That is not cleaning it up, but making it worse. The check if the mods are enabled before turning them off is there to prevent problems caused my mods being disabled twice.

**Comment by ghost** [comment]
Ah I see. Maybe you could make a function to disable mods in an array?

**Pull request closed without merging**